R/tune_bayes.R

```diff
@@ -260,7 +262,7 @@ tune_bayes.workflow <-
 tune_bayes_workflow <-
   function(object, resamples, iter = 10, param_info = NULL, metrics = NULL,
            objective = exp_improve(),
-           initial = 5, control = control_bayes(), eval_time = NULL, ...,
+           initial = 5, control, eval_time = NULL, ...,
```

Comment:
This argument is always set by the calling function, either `tune_bayes.workflow()` or `tune_bayes.model_spec`.